torch/_dynamo/symbolic_convert.py

```diff
@@ -231,6 +231,18 @@ def _detect_and_normalize_assert_statement(
     return True
 
 
+def jump_check_none(check_is_none: bool):
+    def inner(self: "InstructionTranslatorBase", inst: Instruction):
+        value: VariableTracker = self.pop()
+        is_none = False
+        if value.is_python_constant():
```

Comment:
This doesn't seem right, we could have a graph node that is None.  Not all non-constant types aren't none.

Reply:
Discussed offline, we now remap JUMP_.*_NONE instructions to a sequence of other supported instructions.